src/scenes/AcceleratorRouter/ParticipantProjects/EditView/index.tsx

```diff
@@ -66,185 +87,201 @@ const EditView = () => {
+              onChange={onChangeParticipantProject}
+              value={participantProject?.countryCode}
+            />
+            <RegionSelect
+              id={'region'}
+              label={strings.REGION}
+              onChange={onChangeParticipantProject}
+              value={participantProject?.region}
+            />
+            <LandUseMultiSelect
+              id={'landUseModelTypes'}
+              label={strings.LAND_USE_MODEL_TYPE}
+              onChange={onChangeParticipantProject}
+              value={participantProject?.landUseModelTypes}
+            />
+            <ProjectFieldTextfield
+              id={'numNativeSpecies'}
+              label={strings.NUMBER_OF_NATIVE_SPECIES}
+              onChange={onChangeParticipantProject}
+              value={participantProject?.numNativeSpecies}
+            />
+            <ProjectFieldTextfield
+              id={'applicationReforestableLand'}
+              label={strings.APPLICATION_RESTORABLE_LAND}
+              onChange={onChangeParticipantProject}
+              value={participantProject?.applicationReforestableLand}
+            />
+            <ProjectFieldTextfield
+              id={'confirmedReforestableLand'}
+              label={strings.CONFIRMED_RESTORABLE_LAND}
+              onChange={onChangeParticipantProject}
+              value={participantProject?.confirmedReforestableLand}
+            />
+            <ProjectFieldTextfield
+              id={'totalExpansionPotential'}
+              label={strings.TOTAL_EXPANSION_POTENTIAL}
+              onChange={onChangeParticipantProject}
+              value={participantProject?.totalExpansionPotential}
+            />
+            <ProjectFieldTextfield
+              id={'minCarbonAccumulation'}
+              label={strings.MINIMUM_CARBON_ACCUMULATION}
+              onChange={onChangeParticipantProject}
+              value={participantProject?.minCarbonAccumulation}
+            />
+            <ProjectFieldTextfield
+              id={'maxCarbonAccumulation'}
+              label={strings.MAXIMUM_CARBON_ACCUMULATION}
+              onChange={onChangeParticipantProject}
+              value={participantProject?.maxCarbonAccumulation}
+            />
+            <ProjectFieldTextfield
+              id={'perHectareBudget'}
+              label={strings.PER_HECTARE_ESTIMATED_BUDGET}
+              onChange={onChangeParticipantProject}
+              value={participantProject?.perHectareBudget}
+            />
+            <ProjectFieldTextfield
+              id={'numCommunities'}
+              label={strings.NUMBER_OF_COMMUNITIES_WITHIN_PROJECT_AREA}
+              onChange={onChangeParticipantProject}
+              value={participantProject?.numCommunities}
+            />
+            <ProjectFieldDisplay label={strings.DEAL_STAGE} value={participantProject?.dealStage} />
+            {/* TODO need to know where this is supposed to come from, participant project details or project */}
```

Comment:
These are commented out for now because I need to add the data to the "get project" response in the back end.